app/components/LandingPage.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import ProductNavigation from './ProductNavigation';
+import NavBar from './NavBar';
+
+export default class LandingPage extends Component {
```

Comment:
This does not seem like it will ever be a stateful component, so it would be better defined as a function than as a class.